tests/Common/DataFixtures/Executor/ORMExecutorTest.php

```diff
@@ -135,9 +135,9 @@ private function getMockFixture(): FixtureInterface
         return $this->createMock(FixtureInterface::class);
     }
 
-    /** @return ORMPurger&MockObject */
-    private function getMockPurger(): ORMPurger
+    /** @return ORMPurgerInterface&MockObject */
+    private function getMockPurger(): ORMPurgerInterface
```

Comment:
Can't we use a native type hint here?

Reply:
Yes, updated to use intersection type for every getMock in tests